tests/integration/encode_driver_batching/test_encode_driver_batching.py

```diff
@@ -82,10 +82,10 @@ def document_generator(num_docs, num_chunks, num_chunks_chunks):
         yield doc
 
 
-@pytest.mark.parametrize('request_batch_size', [5, 100, 500])
+@pytest.mark.parametrize('request_batch_size', [8, 16, 32])
 @pytest.mark.parametrize('driver_batch_size', [8, 16, 64])
```

Comment:
reduce also driver batch sizes so that most of the tests the driver batch size is smaller than the request size